core/src/test/scala/org/apache/spark/metrics/sink/GraphiteSinkSuite.scala

```diff
@@ -90,7 +90,7 @@ class GraphiteSinkSuite extends SparkFunSuite {
     }
     assert(e.getErrorClass === "GRAPHITE_SINK_PROPERTY_MISSING")
     assert(e.getMessage ===
-      "[GRAPHITE_SINK_PROPERTY_MISSING] Graphite sink requires 'host' property.")
+      "[GRAPHITE_SINK_PROPERTY_MISSING] Graphite sink requires 'host' property. SQLSTATE: KD000")
```

Comment:
Since you are here, please, use `checkError`.

Reply:
Done